discussions/api.py

```diff
@@ -314,12 +316,20 @@ def get_membership_ids_needing_sync():
         QuerySet: query for the list of database ids for memberships that need to be synced
     """
     # Order by is_member (True before False) and updated_on (most recent first)
-    return PercolateQueryMembership.objects.filter(
+    memberships_quest = PercolateQueryMembership.objects.filter(
         query__source_type=PercolateQuery.DISCUSSION_CHANNEL_TYPE,
         user__profile__isnull=False,
         needs_update=True
     ).order_by('-is_member', '-updated_on').values_list("id", flat=True)
 
+    # if membership is created/modified older then 24 hours, mark it as stale
+    stale_memberships = memberships_quest.filter(updated_on__lt=(datetime.now(tz=timezone.utc) - timedelta(hours=24)))
+
+    if stale_memberships:
+        email_mm_admin_about_stale_memberships(stale_memberships.values_list("id", flat=True))
```

Comment:
We shouldn't be doing this in this method because it's called by a task that's called as often as every minute, so this will quickly get annoying. Also, a failure to send the email would have the side affect of causing an error in any operation that calls this, so it's also not ideal in that regard.

This should at a minimum be a separate celery task that performs the check and sends the email to isolate failure cases of that from this code. It might also be feasible to accomplish this with a Redash alert depending on what @pdpinch needs.

Reply:
@pdpinch Thoughts about alerting medium ?

@rhysyngsun Are we using Redash already in project ?